lib/gibbon/request.rb

```diff
@@ -1,15 +1,17 @@
 module Gibbon
   class Request
-    attr_accessor :api_key, :api_endpoint, :timeout, :proxy, :faraday_adapter, :debug, :logger
+    attr_accessor :api_key, :api_endpoint, :timeout, :open_timeout, :proxy, :faraday_adapter, :debug, :logger
 
     DEFAULT_TIMEOUT = 30
+    DEFAULT_OPEN_TIMEOUT = 60
```

Comment:
I will probably change this to a lower value later. Open timeout ought to be lower than timeout, right?